### PR TITLE
remove use of deprecated `parse_log_for_error` in `RPackage` easyblock

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyblocks/generic/rpackage.py [hunk 1]
```diff
@@ -183,8 +183,16 @@ def check_install_output(self, output):
         """
         Check output of installation command, and clean up installation if needed.
         """
-        errors = parse_log_for_error(output, regExp="^ERROR:")
+        reg = re.compile("^ERROR:", re.I)
+
+        errors = []
+        for line in output.split('\n'):
+            r = reg.search(line)
```
[the remark this answers is not in the document]
Comment (branfosj) [reply easybuild/easyblocks/generic/rpackage.py]: done in 9bb5841

**Comment by branfosj** [comment]
There are test reports using this in https://github.com/easybuilders/easybuild-easyconfigs/pull/21908

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Merging, thanks @branfosj

**Review by lexming — APPROVED** [review]

**Pull request merged**